bundles/turn-combat/lib/CombatController.js

```diff
@@ -127,19 +128,25 @@ module.exports = class CombatController {
 
       p.character.combatData = {};
 
-      if (p.character.isNpc) {
-        this.gameState.MobManager.removeMob(p.character);
-        continue;
-      }
-
-      // at the end of combat put downed players at 1 HP
       if (condition === 'downed') {
-        p.character.setAttribute('health', 1);
-      }
-
-      p.character.emit('combatEnd');
-      Broadcast.sayAt(p.character, '-> The battle is over!');
-      Broadcast.prompt(p.character);
+        if (p.character.isNpc) {
+            this.gameState.MobManager.removeMob(p.character);
+            continue;
+        }
+        else
+        //put downed players at 1 HP
+        {
+            p.character.setAttribute('health', 1);
+        }
+	  }
+	  B.sayAt(p.character, '-> The battle is over.');
+	  if (!p.character.isNpc) {
+	      p.character.queueCommand({
```

Comment:
Interesting approach here. Why not just emit the combat end event directly?

Reply:
Ah, I see this must be the fix for the extra combat messages. I'd put a comment here explaining that. Seems like an odd side effect but glad you figured this one out.

Reply:
Note that this occurs on a ranvier using the standard bundles with turn-combat added. It might not be occurring on a ranvier using the full trpg-skeleton